.github/workflows/unit.yml

```diff
@@ -29,7 +29,9 @@ jobs:
           rustup default ${{ matrix.rust }}
 
       - name: Check cargo test
-        run: cargo test --workspace --all-features --no-fail-fast
+        run: |
+          cargo clean
```

Comment:
same here https://github.com/dfinity/sdk/pull/2513/files#diff-e17ac7ab478a18e9dac875e191cc3e0d754fab62eddddfb05d4619495be927d9L23

Reply:
What I wonder about this is: does that mean that `cargo clean` here is a mistake?

For the e2e case, the prepare-dfx-assets.sh step is complex enough that it would be pretty easy to get a stale result.  but here?

Reply:
I don't know if it's a mistake, but I'm unsure about what's the benefit of running `cargo clean` here. Would love to hear @adamspofford-dfinity thoughts

Reply:
I reverted this change